core/lib/spree/core/controller_helpers/auth.rb

```diff
@@ -48,11 +48,6 @@ def current_ability
           @current_ability ||= Spree::Ability.new(try_spree_current_user)
         end
 
-        def redirect_back_or_default(default)
```

Comment:
Same here, we can't remove this method because we are not sure how/if people are using it. Better to deprecate it.